include/boost/url/impl/url.ipp

```diff
@@ -513,6 +527,13 @@ set_password(string_view s)
     return *this;
 }
 
+url&
+url::
+set_password(pct_encoded_view s)
```

Comment:
Honestly I thought we were doing
```
set_encoded_password( pct_encoded_view )
```
not
```
set_password( pct_encoded_view )
```

Reply:
Your original [comment](https://cpplang.slack.com/archives/C01JR6C9C4U/p1658262764343469) referred to

> I'm open to 3 functions instead of 2 or 1:

```cpp
url& url::set_encoded_X( string_view );
url& url::set_X( string_view );
url& url::set_X( pct_encoded_view const& );
```

I agree with that original view. It makes much more sense semantically, and the other member functions make much more sense:

```cpp
u1.set_user(u2.user());
```

instead of 

```cpp
u1.set_encoded_user(u2.user());
// people will be confused: how are these not symmetrical / can we set encoded with a decoded result
```

and also, of course, the fact that `pct_encoded_view` dereferences to decoded characters.